rules/integrations/aws/defense_evasion_cloudtrail_logging_deleted.toml

```
@@ -86,9 +88,26 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset:aws.cloudtrail and event.provider:cloudtrail.amazonaws.com and event.action:DeleteTrail and event.outcome:success
+event.dataset:aws.cloudtrail
+    and event.provider:cloudtrail.amazonaws.com
+    and event.action:DeleteTrail
+    and event.outcome:success
 '''
 
+[rule.investigation_fields]
+field_names = [
+    "event.action",
+    "event.outcome",
+    "cloud.region",
+    "user_agent.original",
+    "user.name",
+    "related.user",
```

```suggestion
    "user.name",
```
these are both the same for this event so I'd just choose one of them

---

and if the order matters we should probably put the "WHO" first

---

> these are both the same for this event so I'd just choose one of them
Fixed, thank you for identifying that.

> and if the order matters we should probably put the "WHO" first
I am not sure that order matters, it is up to us to make that distinction. I do like the idea of doing the following, granted it will not be applicable in some instances. Thoughts? cc @shashank-elastic 

who (user, target)
what (action, outcome)
where (provider, region) 
how (request, response)

---

I like that general order of events, it'll keep things something consistent across rules.

---

Great, I'll re-order them all and ensure they are who, what, where, how if applicable.